### ISSUE TITLE
Cannot cancel 'Clear Property Value' with Enter

### ISSUE THREAD

**Issue by tznind** [tznind/TerminalGuiDesigner#53]
![image](https://user-images.githubusercontent.com/31306100/180749289-f4ed37d9-69bc-4d0e-aec6-5b5c0a554b4b.png)

When you press keyboard Enter key on the Cancel button it just moves to Ok.  Maybe theres a handled state that needs set?

Also how about it says `Clear Property 'Text'` just so the user is absolutely clear on what is getting nuked

**Comment by BDisp** [comment]
When a button is default the enter key is always used. If the cancel button is selected and his action is needed, then the space key is used.

**Comment by tznind** [comment]
> When a button is default the enter key is always used. If the cancel button is selected and his action is needed, then the space key is used.

Enter or Space both work fine for Cancel.  Default only gets fired on Enter if the currently focused view doesn't respond first.  If Cancel has input focus then Enter will get sent to Cancel (correctly).

This bug was actually that I wasn't setting the original 'launch' keypress to handled in one of the `if` pathways.  So hitting Enter on Cancel would indeed cancel it but immediately relaunch the operation again (so the popup reapeared).  

Fix was in #72

**Comment by BDisp** [comment]
Yes, you are right. I made some confusion. Handled must return always true, no matter what button was used.
